backend/src/relationships/friendships/friendships.service.ts

```diff
@@ -56,15 +57,23 @@ export class FriendshipsService {
     query?: QueryFriendshipDto,
     pageOptions: PageOptionsDto = new PageOptionsDto(),
   ): Promise<PageDto<ResponseFriendshipDto>> {
-    const response = await this.friendshipRepository.findAllPaginated(
-      query,
+    const orderOptions = { id: pageOptions.order };
+    const findOptionsWhere: FindOptionsWhere<Friendship> = {
+      source: query?.sourceId ? { id: +query.sourceId } : {},
+      target: query?.targetId ? { id: +query.targetId } : {},
+      status: query?.status ? query.status : undefined,
```

Comment:
```suggestion
      status: query?.status,
```
Those are strictly equivalent.